cmake/dependencies/PICSAR.cmake

```
@@ -109,7 +109,7 @@ if(WarpX_QED)
     set(WarpX_picsar_repo "https://github.com/ECP-WarpX/picsar.git"
         CACHE STRING
         "Repository URI to pull and build PICSAR from if(WarpX_picsar_internal)")
-    set(WarpX_picsar_branch "24.09"
+    set(WarpX_picsar_branch "6b46dbe083daf30286631c82818d4285c2676371"
```

Since there are no updates to PICSAR-QED [since the last tag](https://github.com/ECP-WarpX/picsar/commits/development/), I would keep a named tag here for self-description:
```suggestion
    set(WarpX_picsar_branch "24.09"
```

---

Oh okay, sounds good to me. We'll do so as well in the future.